### ISSUE TITLE
Windows 7 Qube Can't Connect to the Network After Installing Qubes Windows Tools

### ISSUE THREAD

**Issue by forsakenone9** [QubesOS/qubes-issues#4588]
Even after playing around with the IPv4 configuration (IP, gateway) in the Windows Qube I still have no luck connecting the Internet (1.1.1.1) or pinging another Qube on my system. (sys-firewall and sys-net tested)

I'm using Qubes 4.0 completely up-to-date freshly installed as of yesterday.

Also, I read the readme about the exclamation mark on the network connection icon is a "cosmetic issue" but it's much more than that for me.

Also, also, I installed all the modules offered by the Qubes Windows Tools installer and installed it on dom0 through the testing repo as on Qubes 4.0 it's only at that stage. I'm thinking maybe if I disable a couple one of the modules networking will work because I think I was reading something about it and `qvm-block` but I would rather keep everything enabled if at all possible.

Also, also, also, I just tried seeing if I could connect to the network after booting into Safe Mode with Networking but still no dice.

Current Network Configuration State:
![image](https://user-images.githubusercontent.com/45714818/49687262-41d8dc80-fac6-11e8-9c3e-97dec53e26b9.png)
^ Ping in image above returns a "Destionation host unreachable."

If there are any logs I could provide to help solve this issue I would be happy to comply.

Thank you for your time! :)

**Comment by andrewdavidwong** [comment]
Did networking work before installing Qubes Windows Tools? If so, this is a duplicate of #2263.

**Comment by forsakenone9** [comment]
Good news! I realized some other things in my Qubes was acting up after installing Qubes Windows Tools to my Win 7 VM so I rebooted my Qubes OS, opened Windows, and it's able to connect to the internet now! :D Sorry for the false alarm.

I realize now it's most likely because I was running out of disk space because Qubes just prompted me with a warning in the system tray so I'm going to transfer Windows to a secondary disk to prevent this problem in the future. Would've been nice if it gave the warning a bit before reaching 99.4% of drive's capacity though.

Thank you for the work you've put into this project by the way! Security by isolation is a revolutionary way of securing a system and I'm so glad that it's being brought to the open source community in such an accessible way!
